### PR TITLE
Update DisconnectReason

### PR DESCRIPTION
**Pull request by bobpetrov**
Added blockedNumber 403 and unavailableService 503

### CODE REVIEW

**Comment by Auties00** [comment]
As far as I remember 403 is used also if the login keys are invalid, so I'm not sure whether the name blocked number is appropriate

**Comment by PurpShell** [comment]
> As far as I remember 403 is used also if the login keys are invalid, so I'm not sure whether the name blocked number is appropriate

Renamed to forbidden.

File: src/Types/index.ts [hunk 1]
```diff
@@ -16,14 +16,16 @@ import { SocketConfig } from './Socket'
...
+    connectionLost = 408,
     connectionReplaced = 440,
     timedOut = 408,
-	loggedOut = 401,
+    loggedOut = 401,
     badSession = 500,
     restartRequired = 515,
-    multideviceMismatch = 411
+    multideviceMismatch = 411,
+    blockedNumber = 403,
```
Comment (purpshell) [inline src/Types/index.ts]:
```suggestion
    forbidden = 403,
```

**Review by PurpShell — APPROVED** [review]
looks good

**Review by SheIITear — APPROVED** [review]

**Pull request merged**